### PR TITLE
Translate gbfs.org into French

### PR DESCRIPTION
**Pull request by richfab**
This PR adds French translations for all content on gbfs.org using [gender inclusive language](https://www.noslangues-ourlanguages.gc.ca/fr/cles-de-la-redaction/ecriture-inclusive-lignes-directrices-ressources.html). 

French was prioritized  because gbfs.org will be referenced at conferences in France (and because it's the only language I speak fluently with English ^^). Other languages will follow.

Modules:
- [Simpleen](https://simpleen.io/), an automated translation tool to gbfs.org repo to generate translations (also used for gtfs.org)
- [mkdocs-i18n](https://pypi.org/project/mkdocs-i18n/), a plugin to allow users to select a language on gbfs.org (also used on gtfs.org)

Before | After
-- | --
image | Screenshot 2024-02-22 at 19 05 07
image | image
\+ 8 more pages | 

Note that I could not translate the banner at the top and the footer.

### CODE REVIEW

**Comment by fredericsimard** [comment]
@richfab Did you use Simpleen to do the translations or used the existing files before the design sprint?

**Comment by fredericsimard** [comment]
Cancel my previous comment... we need to use this structure:
```
.
├─ en/
│  └─ index.md
├─ fr/
│  └─ index.md
└─ mkdocs.yml
```
... to remove the confusion in managing files.

File: mkdocs.yml [hunk 1]
```diff
@@ -82,14 +84,29 @@ plugins:
         'about.md': 'participate.md'
   - external-markdown
   - i18n:
-      docs_structure: folder
+      docs_structure: suffix
```
Comment (fredericsimard) [inline mkdocs.yml:87]: We will use `folder` as the structure, we already decided on that. It's a mess otherwise with tons of files with the same name.

**Review by fredericsimard — CHANGES_REQUESTED** [review]
Switch back to `folder`.

File: mkdocs.yml (hunk 1 above)
Comment (richfab) [reply mkdocs.yml]: Done ✅

**Comment by richfab** [comment]
For future reference, these terms need to be manually replaced with gender inclusive language:

- utilisateur
- voyageur
- contributeur
- régulateur
- développeur
- chercheur

And these terms need to be manually replaced:

- dépôt
- facteur de forme
- géo-clôturées
- never translate "GBFS"
- spécification générale des flux de vélos en libre-service
- consommateur (application réutilisatrice)
- point de terminaison

**Comment by fredericsimard** [comment]
@richfab Looks great, just one little visual issue on the Specifications page, the left navigation section cuts off in french, I've seen this before, I'll try to find the issue.

### French:
Screenshot 2024-02-23 at 09 03 26

### English:
Screenshot 2024-02-23 at 09 03 17

**Comment by fredericsimard** [comment]
> For future reference, these terms need to be manually replaced with gender inclusive language:...

I'll make a script for this eventually, safest option. But I wonder at which step? When people commit markdown files and they get edited right then, or after generating the site maybe?

**Comment by richfab** [comment]
> I'll make a script for this eventually

I think simpleen.io glossary should achieve this but it did not work when I tried it 🤷‍♂️

**Comment by fredericsimard** [comment]
> > I'll make a script for this eventually
> 
> I think simpleen.io glossary should achieve this but it did not work when I tried it 🤷‍♂️

We want to move away from Simpleen, or at least not have it as the only option. At the very least using a script would be a second check just in case.

**Review by fredericsimard — APPROVED** [review]
LGTM!

**Comment by fredericsimard** [comment]
@richfab I'm about to go through the backlog of translations for the GTFS website, do you think these changes should be applied as well? Or is this only for GBFS?

> For future reference, these terms need to be manually replaced with gender inclusive language:
> 
> English	Find	Replace with
> user	utilisateur	utilisateur·rice
> traveler	voyageur	voyageur·euse
> contributor	contributeur	contributeur·rice
> regulator	régulateur	régulateur·rice
> developer	développeur	développeur·euse
> researcher	chercheur	chercheur·euse
> And these terms need to be manually replaced:
> 
> English	Find	Replace with
> repository	dépôt	répertoire
> form-factor	facteur de forme	type de véhicules
> geofencing	géo-clôturées	geofencing
> GBFS		GBFS
> General Bikeshare Feed Specification	spécification générale des flux de vélos en libre-service	General Bikeshare Feed Specification
> consumer	consommateur	application réutilisatrice
> endpoint	point de terminaison	URL

**Comment by richfab** [comment]
Gender neutral language should be used on all websites in my opinion, including gtfs.org. Thanks! cc @isabelle-dr

**Comment by fredericsimard** [comment]
@richfab I've added the preferred translations to the EN > FR glossary of the translation tool. We could close this PR.

See: https://github.com/MobilityData/mbd-translation-tool/blob/main/glossaries-for-google-translate-api/mobilitydata-glossary-en-to-fr.csv

**Comment by richfab** [comment]
Thank you @fredericsimard! I believe this PR is already closed 👍

**Pull request merged**